new_frontend/src/pages/school-calendar/index.tsx

```
@@ -41,21 +43,62 @@ function StudentProfileWithFilters({ student }: { student: Student }) {
-      classSection={student.classSection}
       reference_number={student.reference_number}
       custom_division={student.custom_division}
       first_name={student.first_name}
       last_name={student.last_name}
       program_name={student.program_name}
-      isLoading={false}
-      children={<>
-        <button className='tw-w-full tw-bg-secondary tw-text-primary tw-p-2 tw-rounded-xl' onClick={() => navigate('/school-calendar', { state: { school: student.school } })}>{t('show_calendar')}</button>
-      </>}
+      isLoading={classDetailsLoading}
+      children={<div className='tw-flex tw-gap-2 tw-flex-col'>
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)